playbooks/common/openshift-checks/README.md

```diff
@@ -36,8 +28,14 @@ against your inventory file. Here is the step-by-step:
     $ cd openshift-ansible
     ```
 
-2. Run the playbook:
+2. Run the appropriate playbook:
+
+    ```console
+    $ ansible-playbook -i <inventory file> playbooks/common/openshift-checks/pre-install.yml
+    ```
+
+    or
 
     ```console
-    $ ansible-playbook -i <inventory file> playbooks/byo/openshift-preflight/check.yml
+    $ ansible-playbook -i <inventory file> playbooks/common/openshift-checks/health.yml
```

Comment:
Exactly the sort of conversation I opened this PR to have :) 

I had my own concept of what the directories are for but consulting the README:

- `byo` (_Bring Your Own_ hosts) has the most actively maintained
  playbooks for installing, upgrading and performing others tasks on OpenShift
  clusters.
- `common` has a set of playbooks that are included by playbooks in
  `byo` and others.
- others are basically un-maintained / deprecated

`byo` strikes me as `for install`. And specifically for installing when you already have hosts. I guess our preflight checks are a good match for that. Health checks really aren't though. I thought it might make sense to have the various check playbooks in the same place ... but maybe not?